src/app/static/src/app/components/header/UserHeader.vue

```diff
@@ -5,13 +5,13 @@
                 <div class="navbar-header">
                     {{ title }}
                 </div>
-                <router-link id="projects-link" v-if="!isGuest" to="/projects" class="ml-2 pr-2 border-right"
+                <router-link id="projects-link" v-if="!isGuest" to="/projects" class="ml-2 pr-2"
                              v-translate="'projects'"
-                             style="flex:none"></router-link>
-                <file-menu :title="title || ''"></file-menu>
+                             style="flex:auto"></router-link>
                 <span v-if="!isGuest" class="pr-2 mr-2 border-right text-white">
                     <span v-translate="'loggedInAs'"></span> {{ user }}
                 </span>
+                <div v-else style="flex:auto"></div>
```

Comment:
Using this in the header, because we no longer have the "File" button in the header and we may or may not have the user logged in text thought we needed some other div which would expand to fill the horizontal space. So using this to force the separation in the header between the "Naomi" text and the version number drop down.